nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-frontend/src/main/nifi/src/styles.scss

```diff
@@ -317,7 +317,7 @@ $appFontPath: '~roboto-fontface/fonts';
         border-color: $accent-palette-A400;
     }
 
-    button:disabled {
+    button.nifi-button:disabled {
```

Comment:
Can you please combine this with the style rules directly below this? So then it would be:

```
button.nifi-button:disabled {
        color: $primary-palette-200 !important;
        cursor: not-allowed;
        border: 1px solid $primary-palette-200;

        i {
            color: $primary-palette-200 !important;
        }
    }
```

and there would only be one selector for `button.nifi-button:disabled`.

Reply:
Great catch, I'll update it.